scalableminds/webknossos
# production image: explicitly set UID & GID

### URL of deployed dev instance (used for testing):
- https://scmalpineimage.webknossos.xyz/

### Steps to test:
- check that the UID and GID in the container are still `999` by default, e.g. with
  `docker-compose run --rm --entrypoint id webknossos`

------
